rust/libnewsboat/src/utils.rs

```diff
@@ -303,6 +303,60 @@ pub fn strwidth_stfl(rs_str: &str) -> usize {
     }
 }
 
+/// Returns a longest substring fits to the given width.
+/// Returns an empty string if `str` is an empty string or `max_width` is zero.
+///
+/// Each chararacter width is calculated with UnicodeWidthChar::width. If wcwidth() returns None,
```

Comment:
I guess "wcwidth()" should've been replaced by "UnicodeWidthChar::width" here.

Reply:
Uh yes, fixed :slightly_smiling_face: